src/js/base/api.js

```diff
@@ -335,6 +335,8 @@ ripe.Ripe.prototype._requestURLFetch = function(url, options, callback) {
             try {
                 if (contentType.startsWith("application/json")) {
                     result = await response.json();
+                } else if (contentType.startsWith("text/plain")) {

```

Comment:
@BeeMargarida let's go with a less restrictive version (aligned with https://github.com/hivesolutions/yonius/pull/18)
```suggestion
                } else if (contentType.startsWith("text/")) {

```